stm32_loader.py

```diff
@@ -113,8 +115,8 @@ def get(self):
         available_commands = self._read_bytes(response_length)
         self._wait_for_ack()
         return {
-            'version': bl_version,
-            'commands': available_commands
+            b'version': bl_version,
+            b'commands': map(lambda x: b'%.2x' % x, available_commands) 
```

Comment:
These values are supposed to be machine-readable, therefore string conversion should be removed. The old implementation was correct. Also, keys should be str, not bytes.